src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+* The `OpenTelemetryLoggerOptions.AddOtlpExporter` extension no longer
+  automatically set `OpenTelemetryLoggerOptions.ParseStateValues` to `true`.
+  ([#4334](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4334))
```

Comment:
nit: would be nice to indicate /highlight that there will be no visible changes to the output from OTLP.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/4415